packages/frontend/src/commons.ts

```diff
@@ -87,7 +87,7 @@ export const CHAIN_DATA: Record<SupportedChain, ChainData> = {
             logo: markRaw(MuiCeloIcon),
             backgroundColor: "#000",
         },
-        contract: ADDRESS[SupportedChain.CeloAlfajores],
+        contract: ADDRESS[__ENVIRONMENT__][SupportedChain.CeloAlfajores],
```

Comment:
I think going forward we will need a way to improve this as not all environments might have Holesky or Celo Alfajores deployments. Prod for example won't have them.

Actually, can we address this now? CHAIN_DATA should entirely be environment-specific I think (it could be dynamically set based on `__ENVIRONMENT__` and looping over `ADDRESS[__ENVIRONMENT__]`).